src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosDbHeaders.cs

```diff
@@ -14,5 +14,7 @@ public static class CosmosDbHeaders
         public const string RequestCharge = "x-ms-request-charge";
 
         public const string SubStatus = "x-ms-substatus";
+
+        public const string CosmosContinuationTokenSize = "x-ms-continuation-token-size";
```

Comment:
Sorry, one more thing :) we should probably keep the native header name `x-ms-documentdb-responsecontinuationtokenlimitinkb` as specified in [FhirCosmosResponseHandler.cs](https://github.com/microsoft/fhir-server/pull/2971/files#diff-fb06fe5c0d5e2f703b091ad84e1c219b0128e780c448ad9d8569f576e4999653)

Reply:
fixed.